src/fsw/FCCode/MissionManager.cpp

```diff
@@ -378,6 +383,12 @@ void MissionManager::set(sat_designation_t designation)
 void MissionManager::transition_to(mission_state_t mission_state,
                                    adcs_state_t adcs_state)
 {
+    if (mission_state == mission_state_t::standby)
+        adcs_dcdc_fp->set(true);
+    else if(mission_state == mission_state_t::safehold)
+        adcs_dcdc_fp->set(false);
+    // all other transitions shall leave the DCDC's alone
```

Comment:
I would strongly suggest the following change:
```suggestion
   if(mission_state == mission_state_t::safehold)
        adcs_dcdc_fp->set(false);
    // all other transitions shall leave the DCDC's alone
```
Leaving this assertion that the DCDCs are enabled here is extra noise that can only serve to confuse us down the road. We want to strictly enforce that all transitions to standby that require booting the DCDC go through the detumble state otherwise initialization will fail. In my opinion adding this line here can only serve to make this fact tougher to decipher for those reading the code and lead to quite HITL failures that may not happen in HOOTL during mission rehearsals.